doc/user/content/release-notes.md

```diff
@@ -127,7 +127,8 @@ List new features before bug fixes.
   [`mz_kafka_source_statistics`](/sql/system-catalog#mz_kafka_source_statistics),
   containing raw statistics from the underlying librdkafka library.
 
-- Fix a bug that caused a panic when using a query containing `STRING_AGG`
+- Fix a bug that caused a panic when using a query containing
```

Comment:
- Should end in a period
- Adding any kind of detail about when this occurred would be nominally more confidence building in release note readers.

Reply:
sure!